example/App/Container/app.js

```diff
@@ -117,11 +117,11 @@ export default class example extends Component {
 
   render() {
     return (
-      <ScrollView style={{backgroundColor: '#f4f4f4'}}>
```

Comment:
Removing the scrollview here stopped Masonry from rendering images. I moved it into Masonry and that worked.